wincrypto/src/dtls.rs

```diff
@@ -631,12 +631,12 @@ impl Drop for Dtls {
         unsafe {
             if let Some(ctx_handle) = self.security_ctx {
                 if let Err(e) = DeleteSecurityContext(&ctx_handle) {
-                    error!("DeleteSecurityContext on Drop failed: {:?}", e);
+                    debug!("DeleteSecurityContext on Drop failed: {:?}", e);
                 }
             }
             if let Some(cred_handle) = self.cred_handle {
                 if let Err(e) = FreeCredentialsHandle(&cred_handle) {
-                    error!("FreeCredentialsHandle on Drop failed: {:?}", e);
+                    debug!("FreeCredentialsHandle on Drop failed: {:?}", e);
```

Comment:
So what does this error message mean?

Reply:
If the logic is correct that is, I would probably have an assert(FreeCredenti.... .is_ok())

Reply:
The wincrypto is more your domains. Asserts sounds good to me!

Reply:
Changed to expect valid handle